common/serializers/serialization.py

```diff
@@ -13,6 +14,7 @@
 client_req_rep_store_serializer = JsonSerializer()
 multi_sig_store_serializer = JsonSerializer()
 state_roots_serializer = Base58Serializer()
+proof_nodes_serializer = Base64Serializer()
```

Comment:
Please use it for creation of proofs as well

Reply:
Done